content/it/docs/concepts/architecture/controller.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Controllers
+content_template: templates/concept
+weight: 40
+---
+
+{{% capture overview %}}
+
+Nella robotica e nell'automazione, un _control loop_ è un un'iterazione senza soluzione di continuità che regola lo stato di un sistema.
+
+Ecco un esempio di un _control loop_ ovvero un _circuito di controllo_: il termostato di una stanza.
+
+Quando viene impostata la temperatura, si definisce attraverso il termostato lo *stato desiderato*. L'attuale temperatura nella stanza è invece lo *stato corrente*. Il termostato agisce per portare lo stato corrente il più vicino possibile allo stato desiderato accendendo e spegnendo le apparecchiature.
+
+{{< glossary_definition term_id="controller" length="short" >}}
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## Il modello del controller
+
+Un _controller_ monitora almeno una tipo di risorsa registrata in Kubernetes. 
+Questi [oggetti](/docs/concepts/overview/working-with-objects/kubernetes-objects/#kubernetes-objects) hanno una proprietà chiamata *spec* che rappresenta lo stato desiderato. Il o i *controller(s)* per quella risorsa sono responsabili di mantenere lo stato corrente il più simile possibile rispetto allo stato desiderato.
+
+Il _controller_ potrebbe eseguire l'azione relativa alla risorsa in questione da sé; più comunemente, in Kubernetes, un _controller_ invia messaggi all'{{< glossary_tooltip text="API server" term_id="kube-apiserver" >}} che produce utili effetti collaterali. Di seguito troverete esempi per questo scenario.
```

Comment:
I don't like "effetti collaterali" because it has negative acception in Italian, so I'm proposing following translation
```suggestion
Il _controller_ potrebbe eseguire l'azione relativa alla risorsa in questione da sé; più comunemente, in Kubernetes, un _controller_ invia messaggi all'{{< glossary_tooltip text="API server" term_id="kube-apiserver" >}} che a sua volta li rende disponibi ad altri componenti nel cluster. Di seguito troverete esempi per questo scenario.
```

Reply:
totally agree